src/libraries/System.Drawing.Common/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Drawing.Common
+This assembly provides access to GDI+ basic graphics functionality via types such as [`Bitmap`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.bitmap) and [`Font`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.font).
+
+Note that `System.Drawing.Common` is only supported on Windows: https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.drawing.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [ ] [We don't accept refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
+
+## Deployment
+`System.Drawing.Common` is provided as a [NuGet package](https://www.nuget.org/packages/System.Drawing.Common) and part of the `Microsoft.WindowsDesktop.App` shared framework.
```

Comment:
How can I determine if a particular assembly belongs to Microsoft.WindowsDesktop.App or Microsoft.NETCore.App?

Reply:
Look at https://github.com/dotnet/runtime/blob/main/src/libraries/NetCoreAppLibrary.props